src/MainPage.jsx

```diff
@@ -25,10 +24,32 @@ class App extends Component {
       reader: null,
       readerLabel: "",
       registrationCode: "",
-      cancelablePayment: false
+      cancelablePayment: false,
+      chargeAmount: 5100,
+      itemDescription: "Red t-shirt",
```

Comment:
An obvious future refactor would be to introduce collections of items to help test the reader UI more thoroughly.

Reply:
👍